extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -572,6 +609,7 @@ export class AmpStoryPage extends AMP.BaseElement {
       this.checkPageHasAudio_();
       this.checkPageHasElementWithPlayback_();
       this.renderOpenAttachmentUI_();
+      this.renderOpenOutlinkAttachmentUI_();
```

Comment:
I think you could check if the experiment is on here, like:
```
if (this.isPageAttachmentOutlinkV2ExperimentOn_()) {
  this.renderOpenOutlinkAttachmentUI_();
} else {
  this.renderOpenAttachmentUI_();
}
```

This would reduce branching of this codepath.
`renderOpenAttachmentUI_()` wouldn't need to be touched in this PR.